src/common/hooks/EmoteMenuViewStore.jsx

```diff
@@ -0,0 +1,22 @@
+import {useEffect} from 'react';
+import emoteMenuViewStore from '../stores/emote-menu-view-store.js';
+
+export default function useEmoteMenuViewStoreUpdated(shouldUpdate, handleUpdate) {
```

Comment:
did you update usage of this hook? all imports:

1. do not use the new name
2. do not pass the correct args